### PR TITLE
Improving error messages

### LINKED ISSUE #22: When a field's children property is empty, the error is off
When you have this:

```js
      {
        key: 'static',
        required: true,
        children: []
      },
```

Then `zapier validate` will error:

```
Validating project locally.
┌───────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
│ = 1 =                                                                                                         │
│     Property │ App.creates.foo.inputFields[1]                                                                 │
│     Message  │ must not contain deeply nested child fields. One level max.                                    │
│     Links    │ https://github.com/zapier/zapier-platform-schema/blob/v1.0.11/docs/build/schema.md#fieldschema │
└──────────────┴────────────────────────────────────────────────────────────────────────────────────────────────┘
Your app is structurally invalid. Address concerns and run this command again.
```

Instead, it should error that `children` cannot be empty.

### PR DESCRIPTION
**Pull request by BrunoBernardino**
Specifically when a field has an empty array in .children and other kinds of invalid values, showing which field with `</FieldSchema>,</FunctionSchema>` instead of "anything" in `[subschema 0],</FunctionSchema>`.

Fixes #22

### CODE REVIEW

**Comment by kola-er** [comment]
Seen, will review soon.

File: test/readability.js [hunk 1]
```diff
@@ -16,4 +17,58 @@ describe('readability', () => {
...
+          {key: 'field', type: 'string', default: ''},
+        ],
+      },
+    });
+    results.errors.should.have.length(1);
+    // Ideally it would be the commented version, but it would require significant changes in jsonschema
+    // results.errors[0].stack.should.eql('instance.operation.inputFields[0].default does not meet minimum length of 1');
+    results.errors[0].stack.should.eql('instance.operation.inputFields[0] is not exactly one from </FieldSchema>,</FunctionSchema>');
+  });
+
```
Comment (kola-er) [inline test/readability.js:73]: Nice improving the test coverage.

**Review by kola-er — APPROVED** [review]
It fixes the issue of inappropriate error message for the edge case of empty `children` list.
